lib/pyld/jsonld.py

```diff
@@ -4182,9 +4182,9 @@ def _get_inverse_context(self, active_ctx):
                 else:
                     # add an entry for the default language
                     entry['@language'].setdefault(default_language, term)
-                    # add entries for no type and no language
-                    entry['@type'].setdefault('@none', term)
-                    entry['@language'].setdefault('@none', term)
+                # add entries for no type and no language
+                entry['@type'].setdefault('@none', term)
+                entry['@language'].setdefault('@none', term)
```

Comment:
I don't know but I am concerned about verbosity of the resulting document. What do you think about this?